scripts/push-pull-e2e.sh

```diff
@@ -63,6 +63,24 @@ if [ $? != 0 ]; then
     exit 1
 fi
 
+$CONFTEST push localhost:5000/testdataonly -p '' -d examples/data/exclusions
+if [ $? != 0 ]; then
+    echo "ERROR PUSHING BUNDLE"
+    exit 1
+fi
+
+$CONFTEST pull localhost:5000/testdataonly -p testdataonly
+if [ $? != 0 ]; then
+    echo "ERROR PULLING BUNDLE"
+    exit 1
+fi
+
+$CONFTEST verify -p '' -d testdataonly/examples/data/exclusions
```

Comment:
(optional) at first glance, this returns the following
```shell
0 tests, 0 passed, 0 warnings, 0 failures, 0 exceptions, 0 skipped
```
since it's not actually verifying a policy with tests, MB the message could be transformed into a form like `error loading data bundles`

Reply:
The output indicates that there are no policies, which is expected because the command uses a bundle that only has data. I'm not sure I fully understand your comment. Could you clarify it a bit?

Reply:
what I was suggesting is changing the 
`echo "POLICIES WERE NOT SUCCESSFULLY VERIFIED"` to `echo "ERROR LOADING DATA BUNDLES"`
because verify is being used to execute tests written against rego policies. but in your tests, I see you're using it to validate that the data folder is present

Reply:
Yes, that makes sense. Done. Thanks.